FutureNorthants/northants-website
# Add option to show breadcrumb in hero image

Added a checkbox in HeroImage to 'showBreadcrumbs'.

## To Test
- Checkout this branch and run `./drush.wrapper @docker cim`
- Edit a ServiceLandingPage and add a Hero Image, then check the 'Show breadcrumb' checkbox, then publish the page
- Use the below graphql query and ensure showBreadcrumb is true
- Edit the page and uncheck the 'Show breadcrumb' checkbox, then publish the page
- Use the below graphql query and ensure showBreadcrumb is false

```graphql
{
  route(path: "/page-to-test") {
    __typename
    ... on DrupalNodeRoute {
      __typename
      node {
        title
        ... on ServiceLandingPageNode {
          title
          heroImage {
            title
            isBoxed
            imageLarge
            imageSmall
            imageAltText
            showBreadcrumb
          }
        }
      }
    }
  }
}
```
